test/functional/eval/api_functions_spec.lua

```diff
@@ -1,11 +1,33 @@
 local helpers = require('test.functional.helpers')(after_each)
 local Screen = require('test.functional.ui.screen')
 local lfs = require('lfs')
+local mpack = require('mpack')
 local neq, eq, execute = helpers.neq, helpers.eq, helpers.execute
 local clear, curbufmeths = helpers.clear, helpers.curbufmeths
 local exc_exec, expect, eval = helpers.exc_exec, helpers.expect, helpers.eval
 local insert = helpers.insert
 
+local read_mpack_file = function(fname)
```

Comment:
I think all these additions does not belong in `eval/`, rather in `api/vim_spec.lua` or perhaps better a new `api/compatibility_spec.lua` .